src/components/pages/FAQPage/FAQPage.story.tsx

```diff
@@ -19,13 +18,12 @@ const story = storiesOf("Pages | FAQPage", module)
 // Mobile navigation menu
 story.add("Component", () => (
     <FAQPage
-        blockLabelList={blockLabelList}
         emailAddress={data.contact.address}
```

Comment:
`{text("Email Address", "x@x.com")}`

Reply:
Are these always supposed to be dynamic so you can test how the UI looks if you change the text?

Reply:
Most of the time it should be dynamic. Only if you're 100% that the text will never ever change, it doesn't need to be dynamic.  For example, today I was working with a component that no matter where is displayed, it should always show the same text, in that case, it wasn't worth making the text dynamic. 

Since we're working on a Multilingual website there a lot of chances that most of the text will be dynamic.

Reply:
>Since we're working on a Multilingual website there a lot of chances that most of the text will be dynamic.

Damn, I totally forgot about that lol. That's a really good point.